### DISCUSSION TITLE
Clarifications

### DISCUSSION THREAD

**Discussion by ykaich** [geeksblabla/blanat#78] [Q&A] [answered]
Hello, I have a few questions, please:

1- Can we submit multiple times using the same programming language?
2- Is parallel processing permitted?
3- Regarding the input examples, there's a constraint on the ordering of cities (e.g., cityA, ..., cityA, cityB, ..., cityB, cityC, ...). Is this ordering always the case, or can cities appear in any order?

**Comment by aboullaite** [comment, marked as answer]
Hi,

1. You can update you submission as mich as you want. But we expect it to be performant and tested

2. Yes. Feel free to add any techniques that make the code runs faster

3. Order is not guaranteed
